src/geocat/viz/taylor.py

```diff
@@ -423,14 +428,29 @@ def add_xgrid(self,
                            linewidth=linewidth,
                            **kwargs)
 
-    def add_ygrid(self,
+    def add_xgrid(self,
                   arr: typing.Union[xr.DataArray, np.ndarray, list, float],
                   color: str = 'lightgray',
                   linestyle=(0, (9, 5)),
-                  linewidth: int = 1,
+                  linewidth: float = 0.5,
                   **kwargs):
-        """Add gridlines (radii) to the Y axis (standard deviation) specified
-        by array *arr*
+        """Add gridlines to the correlation axis specified by array *arr*.
+
+        This function will be deprecated in favor of `add_corr_grid()`
+        """
+
+        return self.add_corr_grid(arr, color, linestyle, linewidth, **kwargs)
+
+    def add_stdev_grid(self,
```

Comment:
Pandas, numpy, and scipy all use "std" as the abbreviation for standard deviation.  Thoughts on using that instead for consistency with the broader ecosystem?

Reply:
Okay that's good to know, We should use it too then.

Reply:
Will be done in my upcoming push